examples/persisted-store/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@urql/example-using-persisted-store",
+  "version": "1.0.0",
+  "private": true,
+  "license": "MIT",
+  "scripts": {
+    "start:server": "node ./src/server/index.js",
+    "start:client": "webpack-dev-server --hot --inline",
```

Comment:
Not really worth changing now but I've been using parcel recently and it would be a prime candidate for small examples like this!